poc/vdaf_prio3.py

```diff
@@ -323,14 +348,15 @@ def joint_rand_seed(Prio3, k_joint_rand_parts):
         )
 
     @classmethod
-    def joint_rand(Prio3, k_joint_rand_seed):
+    def joint_rands(Prio3, k_joint_rand_seed):
         """Derive the joint randomness from its seed."""
+        binder = b'' if Prio3.PROOFS == 1 else byte(Prio3.PROOFS)
```

Comment:
This imposes an upper bound on the value of `Prio3.PROOFS`. I think we should note this upper bound where `PROOFS` is first declared.

Reply:
Yes. We check this upper bound in `Prio3SumVecWithMultiproof.is_recommended`.  I add comments at first declaration as well.
Having said that, I am not sure what `PROOFS` we will put in the draft? I suppose it depends on robustness analysis. In any cases, due to computation cost, I would imagine > 255 proofs is prohibitive for most use cases.